tools/cosmovisor/cmd/cosmovisor/show_upgrade.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func NewShowUpgradeInfoCmd() *cobra.Command {
+	showUpgradeInfo := &cobra.Command{
+		Use:          "show-upgrade-info <path to executable>",
```

Comment:
The goal of the issue wasn't to pass an upgrade-info location, but to get it from the cosmovisor node config. We have the root of the node, so we can know the path of the executable. Taking an argument doesn't close the issue.